From: Juergen Borleis <jbe@pengutronix.de>
Date: Fri, 25 Aug 2017 15:14:32 +0200
Subject: [PATCH] buildsystem: using macros this way is bogus

'cwautomacros' is a nice, but broken way to use autotools. Remove the
dependency to this external package.

Signed-off-by: Juergen Borleis <jbe@pengutronix.de>
---
 ChangeLog         | 0
 Makefile.am       | 1 -
 configure.ac      | 3 ---
 tilde/Makefile.am | 1 -
 4 files changed, 5 deletions(-)
 create mode 100644 ChangeLog

diff --git a/ChangeLog b/ChangeLog
new file mode 100644
index 000000000000..e69de29bb2d1
diff --git a/Makefile.am b/Makefile.am
index 84176c3f45ef..d9d8115a6758 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,5 +1,4 @@
 AUTOMAKE_OPTIONS = no-texinfo.tex
-ACLOCAL_AMFLAGS=@ACLOCAL_CWFLAGS@
 
 SUBDIRS = tilde .
 
diff --git a/configure.ac b/configure.ac
index 172e3453a553..0c782b45455d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -5,9 +5,6 @@ AM_INIT_AUTOMAKE
 AM_CONFIG_HEADER(config.h)
 AM_MAINTAINER_MODE
 
-dnl Because we use cwautomacros.
-CW_AUTOMACROS
-
 which_cv_iberty=yes
 AC_ARG_ENABLE(iberty,
   [  --disable-iberty        do not try to find and use the iberty library],
diff --git a/tilde/Makefile.am b/tilde/Makefile.am
index 4947448e00d5..1a72615bafc6 100644
--- a/tilde/Makefile.am
+++ b/tilde/Makefile.am
@@ -1,4 +1,3 @@
-ACLOCAL_AMFLAGS=@ACLOCAL_CWFLAGS@
 AUTOMAKE_OPTIONS = foreign
 noinst_LIBRARIES = libtilde.a
 libtilde_a_SOURCES = tilde.c shell.c tilde.h ansi_stdlib.h bashansi.h xmalloc.h
